assets/queries/cloudFormation/ec2_network_ACL_deny_rules/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::EC2::NetworkAclEntry"
+  action := resource.Properties.RuleAction
+  cidr := resource.Properties.CidrBlock
+  unvalidCidr := "0.0.0.0/0"
+  not contains(cidr,unvalidCidr)
```

Comment:
simplify this block by replacing lines 8 and 9 with:
`not contains(cidr, "0.0.0.0/0")`